pkg/patch/patch.go

```diff
@@ -11,16 +11,25 @@ import (
 const (
 	ecrSecret        = `{"name": "appmesh-ecr-secret"}`
 	create           = `{"op":"add","path":"/spec/%v", "value": [%v]}`
+	createElement    = `{"op":"add","path":"/spec/%v", "value": %v}`
 	add              = `{"op":"add","path":"/spec/%v/-", "value": %v}`
 	createAnnotation = `{"op":"add","path":"/metadata/annotations","value":{"%s": "%s"}}`
 	updateAnnotation = `{"op":"%s","path":"/metadata/annotations/%s","value":"%s"}`
+
+	// we don't want to make this configurable since customers shouldn't relies on
+	// this feature to set a fsGroup for them. This feature is just to protect innocent 
+	// customers that are not aware of the limitation of iam-for-service-accounts:
+	// https://github.com/aws/amazon-eks-pod-identity-webhook/issues/8
+	// Customer should set fsGroup on pod spec directly If a specific fsGroup is desired.
```

Comment:
Nit:
```
	// We don't want to make this configurable since users shouldn't rely on this
	// feature to set a fsGroup for them. This feature is just to protect innocent 
	// users that are not aware of the limitation of iam-for-service-accounts:
	// https://github.com/aws/amazon-eks-pod-identity-webhook/issues/8
	// Users should set fsGroup on the pod spec directly if a specific fsGroup is desired.
```

Reply:
neat :D changed 🤣

Reply:
:shipit: